### ISSUE TITLE
Will you continue this precious guide ?

### ISSUE THREAD

**Issue by voducngh** [AktasC/Hackintosh-Elitebook-840-G2-Broadwell#2]
Thanks for starting the guide.
I have the same hardware, but find out there's too much noise.
Could not make the plain EliteBook 840 G2 installing Clover or OpenCore.

I'm asking for your help

**Comment by AktasC** [comment]
Sorry for replying so long after you opened the issue, I was busy with school

The scene evolved so fast the last months, I'm currently reading all the guides available as fast as possible
OpenCore seems to be the future of Hackintosh but it also looks trickier to setup.
My advice would be to go with OC instead of Clover

I will update the guide and change its format to make it easier to read :smile:

**Comment by AktasC** [comment]
You can follow the advancement of the Opencore version here :
https://github.com/AktasC/Hackintosh-Elitebook-840-G2-Broadwell/tree/opencore   

There's also a PR you can follow here : #3 

To be honest, I'm Frankensteinin' the hell out of this laptop and I don't know if it'll survive :thinking:   
if it does, we'll have a brand new and complete guide very soon :rocket:

**Comment by AktasC** [comment]
Hey, some updates :    
Seems like the OpenCore scene is a fucking joke.    
They claim to have better compatibility, perform better and have the latest advancements in the domain but they clearly point out **`"A default config.plist will not be provided.**"` *everywhere*   
**Why ?** Because nobody fucking cares about others, the whole scene is full of cowards and script kiddies, they want you to suffer and maybe break/brick some shit while insuring they won't get sued by Apple.   
The only people knowing what they do are Acidanthera's people and some devs from TonyMacX86's.    

Example to better understand :  
[Any Ryzen CPU needs the exact same basic patches and config.plist](https://khronokernel-2.gitbook.io/opencore-vanilla-desktop-guide/amd-config.plist/amd-config).    
**Regardless of whether you're using a Ryzen 5 / 7 / 9**.   
STILL, they won't provide shit and will, instead, point you some bullshit guide that's full of copy/paste and mistakes which you could probably find a lot of clone all over the internet.  
Thanks people. You're all a bunch of fucking geniuses.    

TL;DR : OpenCore's community is a fucking joke, If you don't need Catalina don't even bother.   
I'm gonna update the Clover guide ASAP.

**Comment by AktasC** [comment]
I got Clover to work properly, installed Mojave and ran it for a day without an issue.   
But for some reason, whenever I try to get from Clover to OpenCore it doesn't work.    

Still investigating this whole config.plist transformation, then I'll cleanup kexts and patches not needed for our hardware and it should be done !

**Comment by AktasC** [comment]
Some updated for this week :

I'm getting ACPI errors but OpenCore boots on the HFS+ MacOS Recovery drive.    
I ned more time to properly convert and cleanup all the ACPI patches from Clover to OpenCore. 

EDIT : It doesn't work but feels like I've advanced a bit, I can see the Apple Logo and the loading bar but it keeps rebooting at 60-70%.

**Comment by voducngh** [comment]
Keep it going.
I have 2 840G2 dedicated that i'm trying to put 10.15.3 or .4 on it.
I unsuccessfully use Clover. So may be OC will be simpler.
Also, the non-access to a MacOS forced me to use a lot of manipulations.

On OC, I can't even pass the boot menu who offer to 1) reset NVRAM.
I keep trying

**Comment by AktasC** [comment]
Got a lot on my plate atm, sorry people but I won't be able to continue this guide for some time. :disappointed: 

I'm still "on it" though, made OC work on my desktop in like an hour whereas _this_ took me a week.  
But I promise I'll publish a finished and simple enough guide before destroying / selling this laptop !

Still, I won't say no to anybody with en Elitebook 840 G1 whom tries to help.  
Any logs and advancements are welcome :+1:   
Simply reply here and I'll contact you
